locksmith/src/operations/wedlocksOperations.ts

```diff
@@ -0,0 +1,80 @@
+import fetch from 'cross-fetch'
+import * as Normalizer from '../utils/normalizer'
+import { UserTokenMetadata } from '../models'
+import config from '../../config/config'
+
+type Params = {
+  [key: string]: any
+}
+
+type Attachment = {
+  path: string
+}
+
+/**
+ * Function to send an email with the Wedlocks service
+ * Pass a template, a recipient, some params and attachements
+ * @param template
+ * @param failoverTemplate
+ * @param recipient
+ * @param params
+ * @param attachments
+ * @returns
+ */
+export const sendEmail = async (
+  template: string,
+  failoverTemplate: string,
+  recipient: string,
+  params: Params = {},
+  attachments: Attachment[] = []
```

Comment:
A nit pick, I prefer an object as an argument when there are too many of these positional argument. 

It will also be easier to build the payload that way so not hugely repetitive.